### PR TITLE
Investments fly tour

### PR DESCRIPTION
**Pull request by jorgeatgu**
Closes https://github.com/PopulateTools/issues/issues/769

## :v: What does this PR do?

Add a new map to show investments in the city.

## :mag: How should this be manually tested?

Visit investments and click in Tour Virtual.

### After this PR

map-fly

## :shipit: Does this PR changes any configuration file?

No

(Changes in these files might need to update the role in Ansible)

## :book: Does this PR require updating the documentation?

- [ ] new site template? Yes, `minimal.html.erb`

### CODE REVIEW

File: app/javascript/gobierto_investments/webapp/components/Map.vue [hunk 1]
```diff
@@ -73,7 +80,8 @@ export default {
       geojsons: [],
       geojsonOptions: {},
       item: null,
-      center: [40.199867, -4.0654947] // Spain center
+      center: [40.199867, -4.0654947], // Spain center
+      titleButton: 'Tour Virtual'
```
Comment (ferblape) [inline app/javascript/gobierto_investments/webapp/components/Map.vue]: Could you translate this string too?

File: app/javascript/gobierto_investments/webapp/components/MapTour.vue [hunk 2]
```diff
@@ -0,0 +1,197 @@
+<template>
+  <div class="map-flyto-container">
+    <div class="map-flyto-header">
+      <div class="site_header_image">
+        <a href="#">
```
Comment (ferblape) [inline app/javascript/gobierto_investments/webapp/components/MapTour.vue]: This link is broken and the src of the image is hardcoded. If you don't know how to get this dynamic ask @entantoencuanto

File: app/views/gobierto_investments/layouts/application.html.erb [hunk 3]
```diff
@@ -10,4 +10,8 @@
   data-turbolinks="false"
 <% end %>
 
+<% if action_name == "tour" %>
```
Comment (ferblape) [inline app/views/gobierto_investments/layouts/application.html.erb]: @entantoencuanto this isn't necessary anymore

File: app/javascript/gobierto_investments/webapp/components/MapTour.vue [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+    reloadTour() {
+      clearTimeout(this.activeTour)
+      this.activeCardId = 0
+      this.cardsOnScreen(this.activeCardId)
+    },
+    backInvestments() {
+      this.$router.back({ name: "home" });
+    },
+    navTo(item) {
+      this.$router.push({ name: "project", params: { id: item } });
```
Comment (ferblape) [inline app/javascript/gobierto_investments/webapp/components/MapTour.vue:223]: Not sure if this is working well, if you visit a project (i.e http://mataro.gobierto.test/inversiones/proyectos/72) you see it with different styles that if you clik in the project card of the virtual tour.

**Review by ferblape — CHANGES_REQUESTED** [review]

**Comment by ferblape** [comment]
@entantoencuanto could you review the failure in the tests with I18n? The strings should be skipped in the i18n-config.yml

**Comment by entantoencuanto** [comment]
> @entantoencuanto could you review the failure in the tests with I18n? The strings should be skipped in the i18n-config.yml

It's ok now, all translations are in use

File: app/javascript/gobierto_investments/webapp/components/MapTour.vue (hunk 4 above)
Comment (jorgeatgu) [reply app/javascript/gobierto_investments/webapp/components/MapTour.vue:223]: Done.

File: app/javascript/gobierto_investments/webapp/components/Map.vue (hunk 1 above)
Comment (jorgeatgu) [reply app/javascript/gobierto_investments/webapp/components/Map.vue]: We need an array with coordinates[lat/long] @Crashillo

**Comment by jorgeatgu** [comment]
You can check and test @ferblape

**Pull request merged**